docs/troubleshooting/index.mdx

```diff
@@ -35,6 +37,48 @@ Alpine is missing some dependencies that [are required](https://github.com/datre
 Adding the following command to the CI should fix this: `apk add curl bash`
 This will add curl and bash commands which our install scripts require in order to run.
 
+---
+
+## Schema validation failures
+
+If you get the following message in your CLI: 
+```terminal
+❌  k8s schema validation error: could not find schema for [KIND]  
+``` 
+It means that the resource's schema which is used to perform the schema valiation was not found, and it can be caused by any of the following:
```

Comment:
It means that the resource's schema was not found.
This can be caused by any of the following reasons: